### PR TITLE
improve helm chart image tag fields

### PR DESCRIPTION
**Pull request by nuzz**
# Changes

Improving the helm chart image tag fields to conform to pattern used in most stable charts.

# Submitter Checklist

- [ ] Includes tests (if functionality changed/added)
- [ ] Includes docs (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md#commit-messages)

### CODE REVIEW

**Comment by waveywaves** [comment]
@jakalkhalili can you PTAL ?

File: chart/jenkins-operator/templates/operator.yaml [hunk 1]
```diff
@@ -1,59 +1,133 @@
-apiVersion: apps/v1
```
Comment (tomaszsek) [inline chart/jenkins-operator/templates/operator.yaml:1]: I think you made a mistake. Here should be a deployment of the operator.

File: chart/jenkins-operator/Chart.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 appVersion: "0.4.0"
 description: Kubernetes native operator which fully manages Jenkins on Kubernetes
 name: jenkins-operator
-version: 0.3.1
+version: 0.3.2
```
Comment (tomaszsek) [inline chart/jenkins-operator/Chart.yaml:5]: Please revert this change we will update it on my own.

**Comment by tomaszsek** [comment]
Outdated, closing.
